plugins/gfm/gfm-all-module-page/src/main/kotlin/org/jetbrains/dokka/gfm/allModulesPage/GfmAllModulesPagaPlugin.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.dokka.gfm.allModulesPage
+
+import org.jetbrains.dokka.allModulesPage.AllModulesPagePlugin
+import org.jetbrains.dokka.base.DokkaBase
+import org.jetbrains.dokka.gfm.GfmPlugin
+import org.jetbrains.dokka.plugability.DokkaPlugin
+
+class GfmAllModulesPagePlugin : DokkaPlugin() {
+
+    private val gfmPlugin by lazy { plugin<GfmPlugin>() }
+
+    private val dokkaBase by lazy { plugin<DokkaBase>() }
+
+    private val allModulesPagePlugin by lazy { plugin<AllModulesPagePlugin>() }
+
+    val locationProvider by extending {
+        (dokkaBase.locationProviderFactory providing GfmMultimoduleLocationProvider::Factory
+                override listOf(allModulesPagePlugin.multimoduleLocationProvider, gfmPlugin.locationProvider)
+                applyIf { modules.size > 1 })
```

Comment:
I know this condition is the same as in base multimodule but i really dont like it. I may be mistaken but no one is preventing you from running multimodule task on single module